_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1246,15 +1246,14 @@ void CUQPMap::Init (mfxU32 picWidthInLumaSamples, mfxU32 picHeightInLumaSamples)
 {
 
     //16x32 only: driver limitation
-    m_width        = (picWidthInLumaSamples   + 31) / 32;
-    m_height       = (picHeightInLumaSamples  + 31) / 32;
-    m_pitch  = (((picWidthInLumaSamples + 31) >> 5) + 63) &~ 63;
-    m_h_aligned = (((picHeightInLumaSamples + 31) >> 5) + 3) &~ 3;
+    m_width        = (picWidthInLumaSamples  + 31) / 32;
+    m_height       = (picHeightInLumaSamples + 31) / 32;
+    m_pitch        = mfx::align_value<mfxU32>(m_width, 64);
```

Comment:
IMO, it makes sense to add utility function like bool aligned(value, alignment) for places like this